.github/pull_request_template.md

```diff
@@ -1,21 +1,26 @@
-<!--- Provide a general summary of your changes in the Title above -->
 
-## Description
-<!--- Describe your changes in detail -->
+### Description
 
-## Related Issue
-<!--- This project only accepts pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
-<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-<!--- Please link to the issue here: -->
+### Related Issue
+ - [ ] If suggesting a new feature or change, please discuss it in an issue first
```

Comment:
Should we ask "If you are proposing new change please raise issue first and then raise PR"